libvips/foreign/dzsave.c

```diff
@@ -1999,14 +1999,14 @@ vips_foreign_save_dz_class_init( VipsForeignSaveDzClass *class )
 		_( "Tile width in pixels" ),
 		VIPS_ARGUMENT_OPTIONAL_INPUT | VIPS_ARGUMENT_DEPRECATED,
 		G_STRUCT_OFFSET( VipsForeignSaveDz, tile_size ),
-		1, 8192, 256 );
+		1, 8192, 254 );
 
 	VIPS_ARG_INT( class, "tile_height", 12, 
 		_( "Tile height" ), 
 		_( "Tile height in pixels" ),
 		VIPS_ARGUMENT_OPTIONAL_INPUT | VIPS_ARGUMENT_DEPRECATED,
 		G_STRUCT_OFFSET( VipsForeignSaveDz, tile_size ),
-		1, 8192, 256 );
+		1, 8192, 254 );
```

Comment:
@jcupitt For these two above here, are they legacy settings? I ask this because of this comment `/* How annoying. We stupidly had these in earlier versions. */` If so and they are used for non-Microsoft DeepZoom images and have no overlap, I should not have changed them to 254 but rather keep them at 256. The goal is to have the majority of tiles be 256x256px and none of them exceed it (or nearest power of two).

On a related note, we should keep in mind that [_Deep Zoom_](https://en.wikipedia.org/wiki/Deep_Zoom) is the product name of the original Microsoft release and generally, the umbrella term would probably better be _image pyramid_ or similar. I am worried that it might be confusing for users to save a _Deep Zoom_ image with _Zoomify_ layout. It might be better to say _image pyramid_ with _Zoomify_ / _Google_ / _Deep Zoom_ layout. Also, as far as I know, only _Deep Zoom_ (DZI) supports tile overlap, which was originally added avoid seams between images due to numerical rounding errors. I don’t think Google Maps or Zoomify pyramids are valid with a non-zero tile overlap value. Is that something we could enforce? Probably too late, but it’d be clearer to name the feature `vips pyramid` or similar and then distinguish between _DZI_ (Deep Zoom Image), Zoomify, etc. Also, consider prefixing the `overlap` parameter with `tile-*` to clarify its meaning :smile: 

Thanks again for making image pyramid generating wicked fast :christmas_tree: